dataactbroker/helpers/script_helper.py

```diff
@@ -93,8 +93,14 @@ def validate_load_dates(arg_start_date, arg_end_date, arg_auto, load_type, arg_d
         logger.error('start_date, end_date, or auto setting is required.')
         raise ValueError('start_date, end_date, or auto setting is required.')
 
-    if start_date and end_date \
-            and datetime.strptime(start_date, output_date_format) >= datetime.strptime(end_date, output_date_format):
+    if (
```

Comment:
Did this come from `black`? We generally try avoiding short lines in conditions unless the grouping makes sense (like the original code). Down the road we can reconsider using black or automatic formatting but it should be all at once instead of piecemeal and inconsistent.

Reply:
No, adding `datetime.datetime.strptime` instead of `datetime.strptime` pushed the line length past 140, so I put each condition on its own line.  I thought it helped with readability.

Reply:
Ah I see, my mistake. I see what you had to work with - it being kinda a long mess regardless. We've generally tried to fit as much as we can on every line to the flake8 width limit with exceptions to specific logical groupings. We can add that to the StyleGuide as beforehand it was just a casual agreement between the two lead devs.

```suggestion
    if start_date and end_date and (datetime.datetime.strptime(start_date, output_date_format)
                                    >= datetime.datetime.strptime(end_date, output_date_format)):
```